.gitignore

```diff
@@ -6,3 +6,4 @@ phpunit.xml
 .idea
 .phpunit.result.cache
 var/
+*/**/.DS_Store
```

Comment:
Should be reverted, same for `.idea`, those should be added to your global .gitignore file

Reply:
why removing them though? Could avoid some unwanted additions, and they don't do harm here, is there any specific reason?

Reply:
I agree with @OskarStark

Reply:
Ok I will revert this and also remove `.idea`, I get that today this is considered the best practice